src/go/rpk/pkg/cli/cmd/acl/user.go

```diff
@@ -138,10 +147,17 @@ acl help text for more info.
 
 			err = cl.CreateUser(cmd.Context(), user, pass, mechanism)
 			out.MaybeDie(err, "unable to create user %q: %v", user, err)
-			fmt.Printf("Created user %q.\n", user)
+
+			userCollection := userCollection{Users: []string{user}}
```

Comment:
I'm not sure we should use the users list here, vs a struct like...
```
struct {
    User string `json:"user"`
}{user}
```

same comment below in delete

Reply:
My original goal here was to reuse the usersCollection struct as much as possible so the output of create/delete/list users was a consistent `users => [ array of users,even if just 1 ]`, but that doesn't really make sense for create/delete as those will only ever return a single affected user, and by returning json that says `users: ["foo"]` the output implies to the consumer that it _could_ be multiple users, so someone writing against this output would write their code accordingly which is a waste of effort. I'll move it to a struct for just a single user as suggested.